src/Operation/Find.php

```diff
@@ -420,7 +429,7 @@ private function createQueryOptions()
             }
         }
 
-        foreach (['collation', 'max', 'min'] as $option) {
+        foreach (['collation', 'max', 'min', 'let'] as $option) {
```

Comment:
```suggestion
        foreach (['collation', 'let', 'max', 'min'] as $option) {
```

This isn't necessary, but it keeps the list alphabetical.